### PR TITLE
Default to desktop images in Cartoons when mobile images are absent

### PR DESCRIPTION
**Pull request by rebecca-thompson**
## What does this change?
This changes the display rules for the Cartoon element. If there are no mobile images, the element will fallback to displaying the desktop images.

## Why?
In the tooling, the element requires at least one desktop image in order to be valid. This means it is possible (and expected) for some cartoons to come through with only one desktop image

## Screenshots

Screenshots don't best capture this behaviour. It is easier to look at the change in the new story created in Storybook, by viewing it at different breakpoints.

### CODE REVIEW

File: dotcom-rendering/src/components/CartoonComponent.tsx [hunk 1]
```diff
@@ -22,6 +22,39 @@ export const CartoonComponent = ({ format, element, switches }: Props) => {
...
+		return (
+			<>
+				<Picture
+					master={image.url}
+					role={element.role}
+					format={{
+						display: format.display,
+						design: format.design,
+						theme: format.theme,
+					}}
```
Comment (mxdvl) [inline dotcom-rendering/src/components/CartoonComponent.tsx:35]: I know this is just moving lines about… but I could not help but notice this one:

```suggestion
					format={format}
```

File: dotcom-rendering/src/components/CartoonComponent.tsx [hunk 2]
```diff
@@ -33,75 +66,18 @@ export const CartoonComponent = ({ format, element, switches }: Props) => {
...
-			</Hide>
-
+			{smallVariant ? (
+				<>
+					<Hide until="desktop">
+						{largeVariant?.images.map(render)}
+					</Hide>
+					<Hide from="desktop">
+						{smallVariant?.images.map(render)}
+					</Hide>
```
Comment (mxdvl) [inline dotcom-rendering/src/components/CartoonComponent.tsx:76]: We tend to design mobile-first, so maybe switching these around would be better? Again, aware that this is simply moving code around.

**Review by mxdvl — APPROVED** [review]
Nice!

Two, small, non-blocking comments about the existing code, not your change, which you are free to ignore!

File: dotcom-rendering/src/components/CartoonComponent.tsx (hunk 2 above)
Comment (rebecca-thompson) [reply dotcom-rendering/src/components/CartoonComponent.tsx]: addressed here: https://github.com/guardian/dotcom-rendering/pull/9460/commits/314f619868672e9ef0641d1e021028be81c3cad6

**Pull request merged**